tools/wireshark/guev1.lua

```diff
@@ -0,0 +1,28 @@
+-- wireshark dissector for GUE variant 1
+
+do
+  local ip = Dissector.get("ip")
```

Comment:
For this and other modules used here - do we need to import them 'properly'?
Linter complains:
tools/wireshark/guev1.lua:4: attempt to index global 'Dissector' (a nil value)
stack traceback:
	tools/wireshark/guev1.lua:4: in main chunk
	[C]: at 0x00404a50
Add @lint-ignore LUA_LUAJIT in a comment above this line to ignore once or @lint-ignore-every LUA_LUAJIT to ignore for the entire file.

Reply:
for wireshark we do not. this one works as is